py/qqa/plots/timeseries.py

```diff
@@ -27,19 +27,19 @@ def generate_timeseries(data_dir, start_date, end_date, hdu, aspect):
         for i,j,y in os.walk(date):
             for file in y:
                 if re.match(r"qa-[0-9]{8}.fits", file):
-                    list_tables += [Table.read(os.path.join(i, file), hdu=hdu)]
-                    #list_tables += [fitsio.read(os.path.join(i, file), hdu, columns=list(QA.metacols[hdu])+[aspect])]
+                    #list_tables += [Table.read(os.path.join(i, file), hdu=hdu)]
+                    list_tables += [fitsio.read(os.path.join(i, file), hdu, columns=list(QA.metacols[hdu])+[aspect])]
```

Comment:
This line fails if a requested metric isn't in a particular file.  e.g. try testing with night 20100306 exposures 1076, 1077, 1159, 1160 for a timeseries of PER_CAMERA MEANDX.  The first two exposures are arc exposures and have the PER_CAMERA HDU (Header Data Unit), while the second two are flavor='ZERO' exposures that don't have PER_CAMERA measurements.  Add wrappers to robustly move on if an input file doesn't have the requested HDU or if it has the HDU but not the requested column.